src/scenes/PlantsDashboardRouter/components/PlantsAndSpeciesCard.tsx

```diff
@@ -17,29 +22,65 @@ import PlantsReportedPerSpeciesCard from './PlantsReportedPerSpeciesCard';
 type PlantsAndSpeciesCardProps = {
   plantingSiteId: number;
   hasReportedPlants: boolean;
+  organizationId: number;
+  projectId?: number;
 };
 
 export default function PlantsAndSpeciesCard({
   plantingSiteId,
   hasReportedPlants,
+  organizationId,
+  projectId,
 }: PlantsAndSpeciesCardProps): JSX.Element {
   const siteReportedPlants = useAppSelector((state) => selectSiteReportedPlants(state, plantingSiteId));
   const theme = useTheme();
   const plantings = useAppSelector((state) => selectPlantingsForSite(state, plantingSiteId));
+  const allPlantings = useAppSelector((state) => selectPlantings(state));
```

Comment:
What is the benefit of passing `state` in here explicitly? I thought that already does that by default?

Reply:
No benefit, just copied from the other selector that was passing the plantingSiteId. Will fix it